tests/nativeapp/test_init.py

```diff
@@ -1,8 +1,6 @@
-from pathlib import Path
 from textwrap import dedent
-from unittest import mock
 
-import pytest
```

Comment:
The reason why removing these imports does not cause the tests to fail in this file is because of an import on line 21 `from tests.testing_utils.fixtures import *` which happens to contain the three imports as well. 
Maybe it would be better to keep these imports explicit in the current file too since if we change any import in `testing_utils.fixtures`, `test_init.py` will also break.

Reply:
This is possible but I want to limit code changes (other than renames) to bare minimum. We can deal with * imports in a followup. I think https://github.com/Snowflake-Labs/snowcli/pull/611 addresses it

Reply:
sgtm, thanks!